src/main/kotlin/inc/evil/d17map/dtos/ReservationResponse.kt

```diff
@@ -25,4 +26,6 @@ data class ReservationResponse(
     val classroom: ClassroomSummary,
     val type: String,
     val numberOfParticipants: Int,
+
+    var recurringId: UUID? = null,
```

Comment:
please add also recurringType and recurringEndDate as optional type so that I can display information about cycle on frontend like that:

![image](https://github.com/user-attachments/assets/46324d65-7d20-4628-90b0-1c9ec98e7ce6)